src/libs/data-utils.js

```diff
@@ -0,0 +1,121 @@
+import _ from 'lodash/fp'
+import filesize from 'filesize'
+import { Fragment } from 'react'
+import { code, div, h, pre } from 'react-hyperscript-helpers/lib/index'
+import Collapse from 'src/components/Collapse'
+import { buttonPrimary, link } from 'src/components/common'
+import { spinner } from 'src/components/icons'
+import Modal from 'src/components/Modal'
+import { TextCell } from 'src/components/table'
+import { Buckets, Martha } from 'src/libs/ajax'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+
+
+const styles = {
+  cell: children => div({ style: { marginBottom: '0.5rem' } }, children),
+  label: text => div({}, text),
+  data: children => div({}, children)
+}
+
+const isFilePreviewable = name => /\.txt$/.test(name) || /\.[ct]sv$/.test(name) || /\.log/.test(name)
+const formatPriceInDollars = price => price < 0.01 ? '< $0.01' : `$${parseFloat(price).toFixed(2)}`
```

Comment:
TOL: I think there are built-in currency formatters in the browser. Not sure if they would actually add any value here, just mentioning.